AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DecalAdded.py

```diff
@@ -164,51 +159,56 @@ def AtomEditorComponents_Decal_AddedToEntity():
         get_opacity_property = decal_component.get_component_property_value(AtomComponentProperties.decal('Opacity'))
         Report.result(Tests.opacity_property_set, get_opacity_property == 0.5)
 
-        # 8. Set Sort Key property on Decal component
+        # 8. Set Normal Map Opacity property on Decal component
+        decal_component.set_component_property_value(AtomComponentProperties.decal('Normal Map Opacity'), value=0.5)
+        get_opacity_property = decal_component.get_component_property_value(AtomComponentProperties.decal('Normal Map Opacity'))
+        Report.result(Tests.normal_map_opacity_property_set, get_opacity_property == 0.5)
```

Comment:
style: consider rewriting the result lines to inline the `decal_component.get_component_property_value(AtomComponentProperties.decal('Normal Map Opacity'))`, that is not use a temporary local variable. the style used by other tests.
this is really a trivial priority refactor thing that doesn't impact function

Reply:
can you elaborate on this? I'm unsure what you mean.

Reply:
```suggestion
        Report.result(
            Tests.normal_map_opacity_property_set,
            decal_component.get_component_property_value(
                AtomComponentProperties.decal('Normal Map Opacity')) == 0.5)
```
you're following the established style of this file though so what you have is absolutely fine.
in other files we aren't using temporary local variables to hold the get_component_property_value since the value isn't typically being used for anything but this result compare.
if you do change to this style you would want to change the style throughout the document to be consistent. It's ultimately not a required change

Reply:
Ah okay that makes sense! Thank you Scott